coderd/httpmw/apikey.go

```diff
@@ -82,6 +82,7 @@ const (
 
 type ExtractAPIKeyConfig struct {
 	DB                          database.Store
+	HandleDormancy              func(ctx context.Context, u database.User) database.User
```

Comment:
reasonable

Reply:
Yeah the passthrough is necessary because we can't import `audit` in `httpmw` since `audit` already imports `httpmw` itself, so I have to result to this jank